### PR TITLE
Comments canvas example

### PR DESCRIPTION
**Pull request by CTNicholas**

### CODE REVIEW

File: examples/nextjs-comments-canvas/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@liveblocks-examples/nextjs-comments-canvas",
+  "description": "This example shows how to build a comments canvas with Liveblocks and Next.js.",
```
Comment (marcbouchenoire) [inline examples/nextjs-comments-canvas/package.json:3]:
```suggestion
  "description": "This example shows how to build a 2D canvas with draggable comments using Liveblocks and Next.js.",
```

File: examples/nextjs-comments-canvas/src/components/CommentsCanvas.module.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  border-radius: 9999px;
+  width: 28px;
+  height: 28px;
+}
+
+.avatar > div {
+  background: lightgray;
```
Comment (marcbouchenoire) [inline examples/nextjs-comments-canvas/src/components/CommentsCanvas.module.css:43]: 🤔 Also this file doesn't have dark mode colors but the root styles do.

**Review by marcbouchenoire — APPROVED** [review]
Good idea to have this as its own example 🙌

**Comment by CTNicholas** [comment]
Thanks @marcbouchenoire! Added dark mode colours.

**Pull request merged**